src/labels/CustomLabels.labels

```diff
@@ -1548,15 +1548,15 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Enablement page header displayed due to missing prerequisites</shortDescription>
-        <value>Enhanced Recurring Donations cannot be enabled</value>
+        <value>Almost there! Just need you to check on a couple of things first . . .</value>
     </labels>
     <labels>
         <fullName>RD2_EnablementDisabledMessage</fullName>
         <categories>Settings</categories>
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Enablement page message displayed due to missing prerequisites</shortDescription>
-        <value>You must enable both Customizable Rollups and Lightning Experience in order to enable Enhanced Recurring Donations.</value>
+        <value>(1) If you're not already using Lightning Experience, click Switch to Lightning Experience at the top of your screen. (2) Enable Customizable Rollups by going to NPSP Settings|Donations|Customizable Rollups. Then return here to the Upgrade to Enhanced Recurring Donations page.</value>
```

Comment:
The message is rendered as is. I have committed a change to render text as `lightning:formattedRichText` and added a new line to separate instructions. Note, the text is center aligned, let me know if it needs to change.
<img width="977" alt="Screen Shot 2019-12-31 at 4 15 14 PM" src="https://user-images.githubusercontent.com/28743286/71636725-02a58e00-2be9-11ea-9b23-d5bc5989a403.png">